experiments/long_runs/land.jl

```diff
@@ -628,7 +629,7 @@ end
 function setup_and_solve_problem(; greet = false)
 
     t0 = 0.0
-    tf = 60 * 60.0 * 24 * 14
+    tf = 60 * 60.0 * 24 * 340
```

Comment:
Any reason why you stop in 340? We have a full 365 days right?

Reply:
I extended this in both our long runs